tests/lib.rs

```diff
@@ -12,6 +12,8 @@ use std::{
     str::FromStr,
 };
 
+const IMPL_VARIANT_COUNT: u8 = 32;
```

Comment:
We could make this a const on the type :)

Reply:
ehe I wasn't even aware of that feature, thanks Thomas! :heart: